refresh-data.sh

```diff
@@ -10,15 +10,41 @@
 set -e
 refresh_dump_name=$1
 
+download_data() {
+	echo 'Downloading fresh production Django database dump...'
+	prod_archive="$refresh_dump_name".gz
+	curl -o $prod_archive $CFGOV_PROD_DB_LOCATION
+	gunzip -f $prod_archive
+}
+
 refresh_data(){
 	echo 'Dropping db'
 	./drop-db.sh
 	echo 'Creating db'
 	./create-mysql-db.sh
 	echo 'Importing refresh db'
 	mysql v1 -u root -p < $refresh_dump_name
-    echo 'Setting up initial data'
-    ./cfgov/manage.py runscript initial_data
+	echo 'Setting up initial data'
+	./cfgov/manage.py runscript initial_data
 }
 
-refresh_data
+# If no argument was passed to the script and the db dump env variable is set
+if [[ -z "$1" && ! -z "$CFGOV_PROD_DB_LOCATION" ]]; then
+	refresh_dump_name='production_django.sql'
+	download_data
+fi
+
+# Only attempt to load data if a db dump file is provided
+if [[ -z "$refresh_dump_name" ]]; then
+	echo 'Please download a recent database dump before running this script:
+
+	./refresh-data.sh production_django.sql
+
+Or you can define the location of a dump and this script will download it for you:
+
+	export CFGOV_PROD_DB_LOCATION=http://bucket-name.s3.amazonaws.com/db_dumps/production_django.sql.gz
```

Comment:
Yeah the real path was so long that it was wrapping to a new line so I truncated it. I'll change this example path to make it clearer that it's fake.